### ISSUE TITLE
Formats for tutorials and screencasts

### ISSUE THREAD

**Issue by sohilpandya** [dwyl/video#32]
So we need to decide upon a format for all the learning videos that we'll be creating. Here is the rough format that has been proposed by @nelsonic in the discussion we've had earlier. 

1. Brief Intro (30 sec)
  - name and what the topic you are covering
2. Show the end result (1 min)
  - What will the user see at the end of the tutorial, this is important as we don't want the user to sit through the whole video if the end result is not what they are looking for
3. Run through the tutorial (4 - 5 min)
4. Retrospective / debrief 
  - tell them what they've achieved and also to find us on GitHub etc. 

This would mean that individual videos are no longer than 6 -7 mins. If a video has more content, we can split it up and have it as parts. (e.g. part - 1, part - 2)

What do you think of the format? Feedback Welcome!

**Comment by Cleop** [comment]
This is the script for the words used in the tachyons-bootstrap video in case anyone wants to use the same wording in the future: https://github.com/dwyl/tachyons-bootstrap/issues/9#issuecomment-311106984

(WELCOME)- Hello, my name is Cleo and I'm a developer at dwyl. This is an introduction to tachyons-bootstrap.

<-- a bit in the middle giving a longer description and why t-b is super cool -->

(WHAT) So let's build something! Go to http://tachyons-bootstrap.dwyl.com, and select the Forms section in the contents. We're going to create our own basic form. (telling the viewer what they're going to make)

(CONCLUSION)- We hope you enjoyed this video please subscribe and check out our other resources on github using the link below.

**Comment by nelsonic** [comment]
The only thing to note/remember is:
![image](https://user-images.githubusercontent.com/194400/27629637-e5378510-5bea-11e7-833d-1b52fbedbe25.png)
(_what's in it for me_?)

There's no need to "_over-sell_" it to the audience, but we should "_frame_" the intro from the perspective of the audience specifically so they know _why_ this is a good decision for _them_:
+ [ ] **The Learner** (_developer_): pitch it in simple language but don't "_dumb it down_" so more _experienced people_ don't feel like it's "not relevant" to them. 
+ [ ] **The Team/Technical Lead** (_or "CTO" / technical decision maker_): explain how the technology gives _specific_ benefits. e.g:
  + [ ] "increase team productivity as features can be built faster.
  + [ ]  reduce time wasted on fixing UI bugs/conflicts because all styles are self-contained
  + [ ] "lower total cost of ownership" over the lifespan of the project.
